components/sync15/src/telemetry.rs

```diff
@@ -630,3 +630,86 @@ mod sync_tests {
         );
     }
 }
+
+/// The Sync ping payload
```

Comment:
A link to https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/data/sync-ping.html wouldn't be out of place here